### DISCUSSION TITLE
Code: Rename PlaygroundEditorTheme to LexicalEditorTheme

### DISCUSSION THREAD

**Discussion by AlessioGr** [AlessioGr/payload-plugin-lexical#31] [Feature Requests]
PlaygroundEditorTheme to LexicalEditorTheme

**Comment by 58bits** [comment]
Nice. The SCSS selector prefixes could also then be renamed like this...
<img width="485" alt="lexical-theme" src="https://user-images.githubusercontent.com/405612/236337972-553279b7-fd39-44cb-b4c1-126f4f6f6e73.png">
